article25.txt

```diff
@@ -1 +1,2 @@
 start deccan article
+Put amilestone in Airlines industry by providing tickets in cheaper price.
```

Comment:
there is no space between a and milestone

Reply:
Noticed.

Reply:
Resolved.